README.md

```diff
@@ -1,18 +1,32 @@
 # CAPI2-BSP (CAPI version 2.0 board support)
-Create CAPI 2.0 board infrastructure including PSL as IP container for various FPGA-cards.
+## What is CAPI 2.0?
+The majority of computational accelerator systems use PCI Express (PCIe) to connect to the main processor
+units via the I/O subsystem. The IBM **C**oherent **A**ccelerator **P**rocessor **I**nterface (CAPI) provides an alternative
+that removes the complexity and overhead of the I/O subsystem to enable higher system performance.
 
-Depends on PSL sources that can be obtained as zip archive from `<TBD>`
+While the first version of CAPI was designed for acceleration in a POWER8 system, its successor CAPI 2.0 is the CAPI version for POWER9 systems.
```

Comment:
While the first version of CAPI was designed for acceleration in a POWER8 system **through the PCIe link**, its successor CAPI 2.0 is the CAPI version for POWER9 systems.

Reply:
Shouldn't the phrase **through the PCIe link** be added to the second half of the sentence which is for POWER9 (in order to distinguish CAPI 2.0 from OpenCapi)?
Or why did you add it to the POWER8 part which does not provide any other option for CAPI anyway?